cmd/snap/cmd_set.go

```diff
@@ -155,6 +161,28 @@ func (x *cmdSet) Execute([]string) error {
 	return nil
 }
 
+func validateJsonKeys(v interface{}) error {
```

Comment:
I still would prefer if this were renamed to `validateJSONKeys`, as I believe this is the current best-practices/what we do now in snapd. Please correct me if there's evidence to the contrary, however.

Reply:
Sorry, I forgot to fix this! Now it's done! Thank you